### PR TITLE
add appointment date to message

### PR DESCRIPTION
**Pull request by sibalzer**

### CODE REVIEW

**Review by paulypeter — DISMISSED** [review]

File: src/impfbot.py [hunk 1]
```diff
@@ -36,13 +37,19 @@ def check_for_slot() -> None:
             log.error("Result is emtpy. (Invalid ZIP Code (PLZ)?)")
         for elem in result:
             if not elem['outOfStock']:
+                local_timezone = timezone(timedelta(hours=1))
```
Comment (paulypeter) [inline src/impfbot.py:40]: I think it should be 2 hours for CEST/MESZ, 1 hour during the winter

Comment (sibalzer) [reply src/impfbot.py:40]: thats right

**Review by paulypeter — APPROVED** [review]
I looked into a more permanent solution, but didn't find one I liked.. I guess this one is fine for a while

**Comment by sibalzer** [comment]
yea same, but we just need the date so it makes no difference if we get dd.mm.YYYY 01:00 during winter.

**Comment by paulypeter** [comment]
You're right, perfect

**Comment by paulypeter** [comment]
@sibalzer What would I have to do to merge this and release it as v3.1.0?

**Comment by sibalzer** [comment]
i'll use the google release please action with semantic commit messages:

 - fix: which represents bug fixes, and correlates to a SemVer patch.
 - feat: which represents a new feature, and correlates to a SemVer minor.
 - feat!:, or fix!:, refactor!:, etc., which represent a breaking change (indicated by the !) and will result in a SemVer major.

**Pull request merged**